docker/manage

```diff
@@ -180,7 +180,7 @@ configureEnvironment() {
   export TAILS_SERVER_URL=${TAILS_SERVER_URL:-"https://tails-dev.vonx.io"}
 
   # frontend
-  export FRONTEND_WEB_PORT=4251
+  export FRONTEND_WEB_PORT=4200
```

Comment:
@esune added the default for Angular to stand up the container for local development, which allows for hot reloading